test/views/CannotClaimViewSpec.scala

```diff
@@ -37,7 +37,9 @@ class CannotClaimViewSpec extends ViewBehaviours {
       val doc = asDocument(applyView)
       assertContainsMessages(doc, "cannotClaim.para1", "cannotClaim.link")
       doc.getElementById("link").attr("href") mustBe "https://www.gov.uk/tax-relief-for-employees"
-    }
 
+      application.stop()
```

Comment:
This is inside the "display page content", it should be outside where the app gets built